package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "milkyway",
-  "version": "0.0.47",
+  "version": "0.0.48",
```

Comment:
i think this should be at 0.0.49 now? might need a rebase

Reply:
Updated!